db/migrate/20170430050223_backfill_library_timestamps.rb

```diff
@@ -0,0 +1,11 @@
+class BackfillLibraryTimestamps < ActiveRecord::Migration
+
+  def change
+    LibraryEntry.where("progress > 0").find_in_batches do |batch|
```

Comment:
maybe I'm wrong, but it looked like `update_in_batches` doesn't take a block, so I wouldn't be able to get references to each entry's `updated_at`